cmd/maya-apiserver/app/command/start.go

```diff
@@ -94,6 +94,7 @@ type CmdStartOptions struct {
 	httpServer *server.HTTPServer
 }
 
+// NewCmdStart start mayaserver
```

Comment:
It looks like `NewCmdStart` only creates a start command object. It does not actually start the server.